values.yaml

```diff
@@ -77,3 +77,74 @@ storage:
   elasticsearch:
     port: 9200
     host: 127.0.0.1
+
+exporter:
+  enabled: false
+  image:
+    repository: registry.ng.bluemix.net/secadvisor_dev/skydive-flow-exporter
+    tag: 12
+  store:
+    bucket: "default"
+    objectPrefix: "default"
+    buffered:
+      max_flows_per_object: 60000
+  write:
+    s3:
+      endpoint: "http://localhost:9000"
```

Comment:
if you are pointing by default to a local minio object store then the minio object store service must be created (for that end you can add an extra container running minio)

Reply:
We suggest to set this value to empty string, add a comment that the user needs to configure it to it's preferred Object Store endpoint, and also configure the credentials. For testing we suggest to add a script in the test directory that will install default minio (with helm install) and also install this skydive helm. Is it acceptable?

Reply:
I want to maintain the property of the skydive helm that it "just works" when using default settings, so once introducing the SA flow exporter I what to maintain this property which requires that we have an available ObjectStore included within our deployment (just is the case with ElasticSearch in which you can use the local instance or override it and point to an external ElasticSearch instance).